website/docs/guides/sl-snowflake-qs.md

```diff
@@ -0,0 +1,878 @@
+      agg_time_dimension: most_recent_order_date
+    description: |
+      semantic model for dim_customers
+    model: ref('dim_customers')
+    entities:
+      - name: customer
+        expr: customer_id
+        type: primary
+    dimensions:
+      - name: customer_name
+        type: categorical
+      - name: customer_type
+        type: categorical
+      - name: first_order_date
+        type: time
+        type_params:
+          time_granularity: day
+      - name: most_recent_order_date
+        type: time
+        type_params:
+          time_granularity: day
+    measures:
+      - name: count_lifetime_orders
+        description: Total count of orders per customer.
+        agg: sum
+      - name: lifetime_spend
+        agg: sum
+        description: Gross customer lifetime spend inclusive of taxes.
+      - name: customers
+        expr: customer_id
+        agg: count_distinct
+
+metrics:
+  - name: "customers"
+    description: "Count of customers"
+    type: simple
+    label: "customers"
+    type_params:
+      measure: customers
+  - name: "customers_with_orders"
+    label: "customers_with_orders"
+    description: "Unique count of customers placing orders"
+    type: simple
+    type_params:
+      measure: customers
+```
+
+This semantic model uses simple metrics to focus on customer metrics and emphasizes customer dimensions like name, type, and order dates. It uniquely analyzes customer behavior, lifetime value, and order patterns.
+
+## Test and query metrics
+
+<!-- The below snippets (or reusables) can be found in the following file locations in the docs code repository) 
+
+https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/snippets/_sl-test-and-query-metrics.md
+-->
+
+<TestQuery />
+
+## Run a production job
+
+<!-- The below snippets (or reusables) can be found in the following file locations in the docs code repository) 
+
+https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/snippets/_sl-run-prod-job.md
+-->
+
+<RunProdJob/>
+
+<details>
+
+<summary>What’s happening internally?</summary>
+- Merging the code into your main branch allows dbt Cloud to pull those changes and builds the definition in the manifest produced by the run. <br />
+- Re-running the job in the deployment environment helps materialize the models, which the metrics depend on, in the data platform. It also makes sure that the manifest is up to date.<br />
+- The Semantic Layer APIs pulls in the most recent manifest and allows your integration information to extract metadata from it.
+</details>
+
+## Set up dbt Semantic Layer
+
+<!-- The below snippets (or reusables) can be found in the following file locations in the docs code repository) 
+
+https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/snippets/_new-sl-setup.md
+-->
+
+<SlSetUp/>
+
+## Connect and query with Google Sheets
+
+<!-- The below snippets (or reusables) can be found in the following file locations in the docs code repository) 
```

Comment:
this isn't something we do but i've added a comment on the snippet location to help future contributions find the snippet when trying to propose changes.